File: patterns/javascript-patterns/pattern60.js
// Pattern 60: ...

//     1
//    * *
//   3 3 3
//  * * * *
// 5 5 5 5 5

const n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = n; j >= 1; j--) {
        if (i >= j) {
            if (i % 2 === 0) {
                row += '* ';
            } else {
                row += i + ' ';
            }
        } else {
            row += ' ';
        }
    }

    console.log(row);
}

File: patterns/python-patterns/pattern60.py
# Pattern 60: ...

#     1
#    * *
#   3 3 3
#  * * * *
# 5 5 5 5 5

n = 5

for i in range(1, n + 1):
    for j in range(n, 0, -1):
        if i >= j:
            if i % 2 == 0:
                print("* ", end="")
            else:
                print(f"{i} ", end="")
        else:
            print(" ", end="")
    print()
